var/spack/repos/builtin/packages/sprng/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install sprng
+#
+# You can edit this file again by typing:
+#
+#     spack edit sprng
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Sprng(AutotoolsPackage):
+
+    # A list of GitHub accounts to
+    # notify when the package is updated.
+    maintainers = ['gmfricke', 'patrickb314']
+
+    homepage = "https://www.sprng.org"
+    url      = "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
+
+    version('5', sha256='9172a495472cc24893e7489ce9b5654300dc60cba4430e436ce50d28eb749a66')
+
+    variant('mpi', default=True, description='Enable MPI support')
+    variant('fortran', default=False, description='Enable Fortran support')
+
+    depends_on('mpi', when='+mpi')
+    depends_on('fortran', when='+fortran')
```

Comment:
Does this actually compile for you? `c`, `cxx` and `fortran` are currently modeled differently from other languages and so we have no provider for them - they are assumed to be generally available. I think you need to remove this line for the package to work.

Reply:
Can you help me understand the issue here? I would like to fix the issues, and make a different pull request as this one  appears to be stagnant. I think this is the only issue I am not sure how to fix.